recording.js

```diff
@@ -152,75 +145,85 @@ exports.startRecording = function(canvases, fileName) {
     )[param];
   };
 
+  /* Determine video information */
+  const fps = 10;
   const height = getMax('height');
   const width = getMax('width');
-  const framePadding = 150;
 
-  /* Grab tracks from canvases */
-  const chunks = new Array();
-  const fps = 10;
-  generateFrame(width, height, framePadding - 50).then(frameCanvas => {
-    const frameStream = frameCanvas.captureStream(fps);
-    const textStream = (
-      _getCanvas(canvases, 'xterm-text-layer') || _getCanvas(canvases, '')
-    ).captureStream(fps);
-    const cursorStream = _getCanvas(
-      canvases,
-      'xterm-cursor-layer',
-    ).captureStream(fps);
-
-    /* Merge tracks into a single stream */
-    let stream;
-    const merger = new VideoStreamMerger({
-      width: width + framePadding * 2,
-      height: height + framePadding * 2,
-      fps,
-      clearRect: false,
-    });
-    merger.addStream(frameStream, {
-      x: 0,
-      y: 0,
-      width: width + framePadding * 2,
-      height: height + framePadding * 2,
-      mute: true,
-    });
-    merger.addStream(textStream, {
-      x: framePadding,
-      y: framePadding,
-      width,
-      height,
-      mute: true,
-    });
-    merger.addStream(cursorStream, {
-      x: framePadding,
-      y: framePadding,
-      width,
-      height,
-      mute: true,
-    });
+  /* Setup recording frame */
+  const { fontSize } = window.store.getState().ui;
+  const frameSize = fontSize * 6;

```

Comment:
Why is this related to the font size? Larger font=larger frame?

Reply:
I did this to avoid situations where the terminal text is super small, but the frame's title bar is giant (like in the README.md example). Let me know if you'd like this to be handled differently 👍

Reply:
oh ok. Makes sense

Reply:
This is a great addition. Didn't even realize such a feature would be possible the way you implemented it. Way to go man 👍